FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  CoreDataFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Shilpa Bansal on 23/02/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+public class CoreDataFeedStore: FeedStore {
+	private let persistentContainer: NSPersistentContainer
+	private let managedContext: NSManagedObjectContext
+	public static let modelName = "FeedStoreDataModel"
+	
+	public init(storeURL: URL?) throws {
```

Comment:
The `storeURL` can be non-optional

```suggestion
	public init(storeURL: URL) throws {
```